Assets/Editor/ModelInspector.cs

```diff
@@ -130,40 +142,40 @@ void Init()
                 m_Settings.meshMultiPreviewMaterial = CreateMeshMultiPreviewMaterial();
                 m_Settings.lineMaterial = CreateLineMaterial();
                 m_Settings.activeMaterial = m_Settings.shadedPreviewMaterial;
-                
+
                 m_Settings.orthoPosition = new Vector3(0.5f, 0.5f, -1);
                 m_Settings.previewDir = new Vector2(-110, 0);
                 m_Settings.zoomFactor = 1.0f;
                 
-                m_Settings.availableDisplayModes = Enumerable.Repeat(true, 7).ToArray();
-                m_Settings.availableUVChannels = Enumerable.Repeat(true, 8).ToArray();
+                m_Settings.skinnedMesh = new Mesh();
```

Comment:
The mesh probably needs to be marked with `HideFlags.HideAndDontSave` (or whatever it's called), since now this will probably create a new mesh, which will get saved into the scene!